### PR TITLE
test(e2e): add e2e tests for notification

### PR DESCRIPTION
**Pull request by maxym-valor**
#### Please provide a link to the associated issue.
Closes

#### Please check whether the PR fulfills the following requirements

- [ ] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [ ] tests for the changes that have been done
- [ ] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [ ] Run npm run build-pack-library and test in external application

Documentation checklist:
- [ ] update `README.md`
- [ ] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [ ] Documentation Examples
- [ ] Stackblitz works for all examples

### CODE REVIEW

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+    it('should check notification in Component as content example', () => {
+        click(asContentExample + button);
+        expect(isElementDisplayed(notificationContainer)).toBe(true, `notification is not opened`);
+        expect(getElementClass(notificationContainer + messageStrip)).toContain('success');
+        click(notificationContainer + button);
+        expect(getText(asContentExample + result)).toContain('Open Button Clicked');
+        click(asContentExample + button);
+        click(notificationContainer + closeButton);
+        pause(1500);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:74]: why the pause here? could we use waitForElDisplayed instead?

File: e2e/wdio/core/tests/notification.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        notificationPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForElDisplayed(notificationPage.title);
+    }, 2);
+
+    it('should check notification with avatar, success indicator and unread title', () => {
+        expect(isElementDisplayed(defaultExample + notificationBody + avatar)).toBe(true, `avatar does not exist`);
```
Comment (Seamoo13) [inline e2e/wdio/core/tests/notification.e2e-spec.ts:52]: could you please rework the selectors a bit. it's hard to read and debug when there are 3+ selectors being merged together.

**Comment by Seamoo13** [comment]
please squash

**Review by Seamoo13 — CHANGES_REQUESTED** [review]
please remove all non-e2e test file changes. (platform table component, etc)

**Comment by maxym-valor** [comment]
> please remove all non-e2e test file changes. (platform table component, etc)

https://github.com/maxym-valor/fundamental-ngx/pull/86

**Pull request closed without merging**